ocs_ci/ocs/resources/pod.py

```diff
@@ -1531,3 +1531,25 @@ def wait_for_pods_to_be_running(
             f"after {timeout} seconds"
         )
         return False
+
+
+def list_of_nodes_running_pods(selector):
```

Comment:
Since this def is in resources->pod, I think you should add the namespace as a parameter to make it more generic.
You can set defaults.ROOK_CLUSTER_NAMESPACE as a default

Reply:
Done